src/App.tsx

```diff
@@ -1,93 +1,78 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useMemo, useState } from 'react';
 
-export const App: React.FC = () => {
-  return (
-    <div className="todoapp">
-      <header className="header">
-        <h1>todos</h1>
+import 'bulma/css/bulma.css';
+import '@fortawesome/fontawesome-free/css/all.css';
+
+import { TodoList } from './Components/TodoList';
+import { Footer } from './Components/Footer';
+import { Header } from './Components/Header';
```

Comment:
Rename the header.tsx to Header.tsx to fix linter errors 
![image](https://github.com/mate-academy/react_todo-app/assets/99403198/98d34119-9243-46d6-8971-8f279945458d)

Reply:
<img width="635" alt="image" src="https://github.com/mate-academy/react_todo-app/assets/70220667/a49a2669-ecb5-43bb-85b3-34a4e8633839">
<img width="341" alt="image" src="https://github.com/mate-academy/react_todo-app/assets/70220667/df77b17c-4af9-49d2-9dd7-f8dcc776f1b0">
still not fixed